### PR TITLE
DEVDOCS-4660: [update] Theme Styles and Properties, clarify force_reload

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-4660]

## What changed?
added text explaining `force_reload`

## Anything else?
Related PRs, salient notes, etc

[DEVDOCS-4660]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4660?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/stencil-docs/reference-docs/styles-and-properties.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ Use the following data types in the `schema.json` file to create configurable op
 
 You can use your widget template settings to define a data type in the schema_translations property of the widget template schema.
 
+<!-- theme: info -->
+> #### `force_reload`
```
Comment (slsriehl) [inline docs/stencil-docs/reference-docs/styles-and-properties.md:7]:
```suggestion
> #### Force reload
```
style guide, no backticks in headings.

File: docs/stencil-docs/reference-docs/styles-and-properties.md [hunk 2]
```diff
@@ -3,6 +3,10 @@ Use the following data types in the `schema.json` file to create configurable op
...
+> Some data types require the `force_reload` field. `force_reload` occurs when Page Builder reloads when the setting defined by `schema.json` changes.
```
Comment (slsriehl) [inline docs/stencil-docs/reference-docs/styles-and-properties.md:8]: Rearrange and unpack the second sentence to describe the `force_reload` event more clearly.  When a data type requires `force_reload`, I'm imagining you define `"force_reload": true` somewhere?

**Comment by bc-traciporter** [comment]
@slsriehl I believe the definition is okay. I am still determining how I would change it. For further information, you can see the slack conversation with Erik Christensen explaining reloading the page vs. reloading the stylesheet. 
https://bigcommerce.slack.com/archives/CCSUP1W1J/p1673294309408249

File: docs/stencil-docs/reference-docs/styles-and-properties.md [hunk 3]
```diff
@@ -3,6 +3,10 @@ Use the following data types in the `schema.json` file to create configurable op
 
 You can use your widget template settings to define a data type in the schema_translations property of the widget template schema.
 
+<!-- theme: info -->
+> #### Force reload
+> Some data types utilize the `force_reload` field. Setting `"force_reload": true` allows Page Builder to reload when the setting defined by `schema.json` changes.
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/styles-and-properties.md:8]:
```suggestion
> Some data types use the `force_reload` field. Setting `"force_reload": true` allows Page Builder to reload when the setting defined by `schema.json` changes.
```

**Review by bc-andreadao — COMMENTED** [review]
added a suggestion that you can take or leave

**Review by slsriehl — APPROVED** [review]
good

**Pull request merged**